os_migrate/plugins/modules/import_workload_prelim.py

```diff
@@ -319,7 +219,9 @@ def run_module():
     # Assume an existing VM with the same name means it was already migrated.
     # Not necessarily true, but force the operator to delete it if needed.
 
-    if len(list(conn.compute.servers(details=False, name=server_name, **module.params['dst_filters']))) > 0:
+    # TODO:FIXME: The following filter returns > 0 and there is no VM
+    # if len(list(conn.compute.servers(details=False, name=server_name, **module.params['dst_filters']))) > 0:
+    if conn.search_servers(server_name):
```

Comment:
The PR is probably just not based on the latest master branch, we shouldn't be reverting this bit i think?

Reply:
Right, I will take this out.